docs/admin/engines/settings.rst

```diff
@@ -377,6 +380,14 @@ engine is shown.  Most of the options have a default value or even are optional.
   * ``ipv4`` set ``local_addresses`` to ``0.0.0.0`` (use only IPv4 local addresses)
   * ``ipv6`` set ``local_addresses`` to ``::`` (use only IPv6 local addresses)
 
+``rate_limit``: optional
+  Limit how many outgoing requests is SearXNG going to send to the engines.
+  Requires redis, see :ref:`redis db`.
+  * ``max_requests`` is the maximum number of requests that will be sent to this
+    engine per interval.
+  * ``interval`` (optional) is the number of seconds before this engine's rate
+    limiter is reset. Defaults to 1 second if unspecified.
```

Comment:
> Am I using the correct formatting in the documentation? The ref link doesn't do anything for me, but nor do any of the other ones in the same page.

Sorry, the issue was already in the "network" paragraph before your paragraph , I fixed it in the master branch:

https://github.com/searxng/searxng/blame/master/docs/admin/engines/settings.rst#L450-L455

In reST paragraphs, lists .. are all "block" elements.  

To start a new block (e.g. a list) a blank line is needed.

Reply:
That worked, thanks.